app/helpers/url_helper.rb

```diff
@@ -0,0 +1,16 @@
+module UrlHelper
+  def path_without_namespace(components, form_builder)
```

Comment:
Assignment Branch Condition size for path_without_namespace is too high. [15.59/15]